### DISCUSSION TITLE
Cannot find Mixed Time-series Chart in 2.0.0

### DISCUSSION THREAD

**Discussion by kenho811** [apache/superset#23277] [Q&A / Help]
Hi all,

I am currently using superset 2.0.0.

I wanted to use the Mixed Time-series chart where I can put two  different charts together (e.g. bar chart + line chart)

However, as you can see in the below screenshot, I can only see `Mixed Chart`.

Mixed Chart only allows me to select 2 different metrics, each one on either side of the y-axis, **from the same underlying dataset**

It does not allow me to juxtapose two charts from two different datasets (something which I guess Mixed time-serties chart would allow me to do )

![image](https://user-images.githubusercontent.com/57944769/222921242-d4da330f-5a26-4989-bb67-39a062be95e1.png)

Are there any feature flags I still need to enable to get the Mixed TIme-series Chart?

The feature flags I have are

```
FEATURE_FLAGS = {
    # ...  # your existing flags
    "ENABLE_TEMPLATE_PROCESSING": True,
    "FAB_API_SWAGGER_UI": True,
    "ALERT_REPORTS": True,
    "GENERIC_CHART_AXES": True,  
    "DASHBOARD_CROSS_FILTERS": True,
}

ALERT_REPORTS_NOTIFICATION_DRY_RUN = False

```

Many thanks

**Comment by nytai** [comment]
It’s the Mixed Chart. With GENERIC_CHART_AXES all the time series specific charts now support other types on the x axis
